config/locales/en.yml

```
@@ -4080,6 +4080,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           select_and_search: "Select filters and click on %{option} to access your data."
         customer_names_message:
           customer_names_tip: "If customer names are hidden for orders you have supplied, you can contact the distributor and ask if they can update their shop preferences to allow their suppliers to view customer names."
+        inventory_overrides_will_be_ignored_warnings: "Note that stock levels reported are from supplier product lists only. If you are using Inventory to manage your stock quantities these values will be ignored in this report."
```

It doesn't look like we have much of a naming convention, but as this is used in the `message` area, it should probably be named "message" instead of "warnings".
In fact I would suggest to include the report name, perhaps matching the file structure like this:

```suggestion
        products_and_inventory:
          all_products:
            message: "Note that stock levels reported are from supplier product lists only. If you are using Inventory to manage your stock quantities these values will be ignored in this report."
```
This would be accessed like:
```
I18n.t("spree.admin.reports.products_and_inventory.all_products.message")
```